bin/gce-pre-align.sh

```diff
@@ -80,10 +78,10 @@ then
 
     if [[ $CONTAINER_IS_RUNNING == 0 && $EXIT_STATUS == 0 ]]
     then
-      OUTPUT_DIR=$2
+      OUTPUT_DIR=$2"/"
       echo "[$(date)] Uploading $OUTPUT_FILE"
       START_TIME=$(date +%s)
-      gcloud compute ssh --zone $MACHINE_ZONE $MACHINE_NAME -- gsutil -o GSUtil:parallel_composite_upload_threshold=150M cp /home/alignment/*.fastq.gz $OUTPUT_DIR
+      gcloud compute ssh --zone $MACHINE_ZONE $MACHINE_NAME -- gsutil -o GSUtil:parallel_composite_upload_threshold=150M cp /home/alignment/*.fastq.gz $OUTPUT_DIR && gcloud compute ssh --zone $MACHINE_ZONE $MACHINE_NAME -- gsutil -o GSUtil:parallel_composite_upload_threshold=150M cp ${OUT_BASE}.list $OUTPUT_DIR 
```

Comment:
We've discussed previously that we wanted four separate buckets to support inputs/outputs at each stage and using rsync here will send the input files to the output directory. The .list file is in the output bucket and the logs are being downloaded to the slurm job working directory. Blindly sending everything to a bucket can bite us in the ass too.